src/core/edgeNetwork/injectSendEdgeNetworkRequest.js

```diff
@@ -67,28 +67,28 @@ export default ({
         return networkResponse;
```

Comment:
The line right above this one is a call to `validateNetworkResponseIsWellFormed`. That's where make sure the response status code is a 2xxx and that the content looks like something we would expect. I question whether `validateNetworkResponseIsWellFormed` is still the right name, but I struggled coming up with a more appropriate name. I'm all 👂👂  if you feel like there's a better name. `validateNetworkResponseIsSuccessful` perhaps?